style/action.css

```diff
@@ -0,0 +1,33 @@
+.action-section {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 180px;
+  background-color: #3882F6;
+  margin: 70px 150px;
+  height: 130px;
+  border-radius: 10px;
+}
+
+.call-to-action-container, .call-to-action-container * {
+  margin: 0;
+}
+
+.call-to-action {
+  color: #F9FAF8;
+  font-size: 24px;
+  font-weight: 700;
+}
+
+.sigh-up-text {
+  color: #E5E7EB;
+  font-size: 18px;
+}
+
+.call-to-action-button {
+  border-color: #F9FAF8;
```

Comment:
No need to change, just a suggestion for next time:
borders have a shorthand value defining:
`border: <size> <style> <color -> defaults to black>`
ex. `border: 1px solid #F9FAF8`

* `border-radius` still need to be a separate property